vue/components/modal.vue

```diff
@@ -80,6 +83,26 @@ body.mobile .modal > .modal-container {
     padding: 20px 10px 20px 10px;
 }
 
+.button.button-close {

```

Comment:
Make this `display: none` by default so we don't break retro-compatibility. Then, RIPE White will override that and use `display: inline-block`